# used for publishing tf of aruco marker
# w.r.t. the camera frame
# ID is required for frame_id
uint8 id
float32[3] rvec
float32[3] tvec